src/App.css

```diff
@@ -139,14 +139,14 @@ table td {
 
 @media only screen and (max-width: 576px) {
  h2 {
-    font-size: 15px !important;
+    font-size: 1.5rem !important;
```

Comment:
1 rem = default font size of the browser
by default 1rem = 16px

here you are converting 1rem => 10px